stable/mongodb-replicaset/values.yaml

```diff
@@ -27,6 +27,21 @@ image:
   tag: 3.6
   pullPolicy: IfNotPresent
 
+# Specs for busybox image
+
+copyConfigImage:
+  busybox:
```

Comment:
Remove `busybox` in between. I don't think that makes sense.

Reply:
Fixed